src/TypeSpec.Extension/Emitter.Csharp/src/sdk-context-options.ts

```diff
@@ -6,5 +6,5 @@ import { CreateSdkContextOptions } from "@azure-tools/typespec-client-generator-
 export const azureSDKContextOptions: CreateSdkContextOptions = {
     versioning: {},
     // TODO: replace with `@hasJsonConverter` when it is available
-    additionalDecorators: []
+    additionalDecorators: ["Azure\\.ClientGenerator\\.Core\\.@clientName"]
```

Comment:
This is for testing. We should add the real Decorators Azure emitter want to emit, e.g. "Azure\.ResourceManager\.$armProviderNamespace"